packages/core/src/hydration/annotate.ts

```diff
@@ -432,10 +429,13 @@ function serializeLView(lView: LView, context: HydrationContext): SerializedView
       continue;
     }
 
-    if (nativeElementsToEventTypes) {
-      // Attach `jsaction` attribute to elements that have registered listeners,
-      // thus potentially having a need to do an event replay.
-      setJSActionAttribute(tNode, lView[i], nativeElementsToEventTypes);
+    // Attach `jsaction` attribute to elements that have registered listeners,
+    // thus potentially having a need to do an event replay.
+    if (nativeElementsToEventTypes && tNode.type & TNodeType.Element) {
```

Comment:
nit (mostly for readability):
```suggestion
    if (nativeElementsToEventTypes && (tNode.type & TNodeType.Element)) {
```

Reply:
done

Reply:
done

Reply:
Reopening this thread, it looks like there was no change made to this part of the code.

Reply:
i think prettier removes those parens